packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -104,6 +112,18 @@ const ContentTable: React.FC<Props> = ({
     [document, base, sorts]
   );
 
+  const [rowObject, setRowObject] = useState<Data>();
```

Comment:
It doesn't seem to be like ''specified(selected)" and "specified data type".
If its type is Data, I think selectedRow,  selectedRowData or selectedData is better.

Reply:
It means specified data, so I will change rowObject to selectedRowData.